database/models.js

```diff
@@ -0,0 +1,78 @@
+const mongoose = require("mongoose");
+const config = require("./config.js");
+const faker = require("faker");
+
+//SCHEMAS
+var gameSchema =  new mongoose.Schema({  
+    name: String,
+    email: String,
+    address: String,
+    bio: String,
+  });  
+  
+  var Game = mongoose.model("Game", gameSchema);
+
+  var relatedContent =  new mongoose.Schema({  
+    name: String,
+    email: String,
+    address: String,
+    bio: String,
+  });  
+  
+  var Related = mongoose.model("Related", relatedContent);
+  
+  //METHODS
+  let saveInfo = () => {
+  for (var i = 0; i < 100; i++) {
+    var example = new Game ({
+        name: faker.name.findName(),
+        email: faker.internet.email(),
+        address: faker.address.streetAddress(),
+        bio: faker.lorem.sentence(),
+     })
+    example.save(err => { 
+      if (err) return console.log(err) });
+    }
+  }
```

Comment:
I'm not sure if this is appropriate separation of concerns. I don't know if you should have your data generator in the models file

Reply:
Part of testing. Will be moved to separate seed file.